modules/iceoryx2/0.5.0.bcr.1/patches/bindgen.patch

```diff
@@ -0,0 +1,72 @@
+diff --git iceoryx2-ffi/ffi/BUILD.bazel iceoryx2-ffi/ffi/BUILD.bazel
```

Comment:
Could you make this a symlink since it's identical?

Reply:
You mean make overlays instead of patch, right?

Reply:
I meant a relative symlink to the same file for the previous version. That makes it more obvious that the file didn't change between versions.

Reply:
updated https://github.com/bazelbuild/bazel-central-registry/pull/3617/commits/f969badd8c8abf039d4469457246ae4a9b46a57c